plugins/amdgpu/criu-amdgpu.proto

```diff
@@ -45,15 +45,15 @@ message device_entry {
 	required bytes private_data = 33;
 }
 
-message bo_entries_test {
-	required uint64	bo_addr = 1;
-	required uint64	bo_size = 2;
-	required uint64	bo_offset = 3;
-	required uint32 bo_alloc_flags = 4;
+message bo_entry {
+	required uint64	addr = 1;
+	required uint64	size = 2;
+	required uint64	offset = 3;
+	required uint32 alloc_flags = 4;
 	required uint32 gpu_id = 5;
-	required uint32 idr_handle = 6;
-	required uint64 user_addr = 7;
-	required bytes bo_rawdata = 8;
+	required uint32 dmabuf_handle = 6;
```

Comment:
There is no point storing the dmabuf handle. You'll get a new handle when the BO is re-created during restore. The restore ioctl will need to return the dmabuf handles for all the VRAM and GTT BOs it created, so that the plugin can use them to restore the BO contents.

Reply:
ACK